hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -221,6 +222,17 @@ public boolean isLockAcquisitionFailed() {
    */
   public final class DeleteCmdWorker implements Runnable {
 
+    private long interval;
```

Comment:
nit: I'll prefer to define it as `Duration` or change variable name to `intervalInMs`.

Reply:
Thanks @hemantk-12  for the comment and review.
I will update soon.